datafusion/sqllogictest/test_files/errors.slt

```diff
@@ -103,10 +103,6 @@ SELECT power(1, 2, 3);
 # Wrong window/aggregate function signature
 #
 
-# AggregateFunction with wrong number of arguments
```

Comment:
I tried to come up with some other query that has an invalid arguments for count(*) but I could not